proto/cosmwasm/wasm/v1/proposal.proto

```diff
@@ -148,3 +148,25 @@ message UnpinCodesProposal {
     (gogoproto.moretags) = "yaml:\"code_ids\""
   ];
 }
+
+// CodeAccessConfigUpdate contains the code id and the access config to be
+// applied.
+message CodeAccessConfigUpdate {
+  // CodeID is the reference to the stored WASM code to be updated
+  uint64 code_id = 1 [ (gogoproto.customname) = "CodeID" ];
+  // InstantiatePermission to apply to the set of code ids
+  AccessConfig instantiate_permission = 4 [ (gogoproto.nullable) = false ];
```

Comment:
Odd that this is 4. Not wrong, just surprising, I assume this was for some backwards compatibility with another struct?

It would be good to label it if so, with something like "2 and 3 are reserved" or such

Reply:
it should be 2 will fix.